dts/arm/nxp/nxp_rt118x.dtsi

```diff
@@ -90,6 +90,72 @@
 		gpio-controller;
 		#gpio-cells = <2>;
 	};
+
+	lpi2c1: i2c@4340000 {
+		compatible = "nxp,imx-lpi2c";
+		clock-frequency = <I2C_BITRATE_STANDARD>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x4340000 0x4000>;
+		interrupts = <13 0>;
+		clocks = <&ccm IMX_CCM_LPI2C0102_CLK 0x70 6>;
+		status = "disabled";
+	};
+
+	lpi2c2: i2c@4350000 {
+		compatible = "nxp,imx-lpi2c";
+		clock-frequency = <I2C_BITRATE_STANDARD>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x4350000 0x4000>;
+		interrupts = <14 0>;
+		clocks = <&ccm IMX_CCM_LPI2C0102_CLK 0x70 8>;
+		status = "disabled";
+	};
+
+	lpi2c3: i2c@2530000 {
+		compatible = "nxp,imx-lpi2c";
+		clock-frequency = <I2C_BITRATE_STANDARD>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x2530000 0x4000>;
+		interrupts = <62 0>;
+		clocks = <&ccm IMX_CCM_LPI2C0304_CLK 0x70 10>;
+		status = "disabled";
+	};
+
+	lpi2c4: i2c@2540000 {
+		compatible = "nxp,imx-lpi2c";
+		clock-frequency = <I2C_BITRATE_STANDARD>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x2540000 0x4000>;
+		interrupts = <63 0>;
+		clocks = <&ccm IMX_CCM_LPI2C0304_CLK 0x80 24>;
+		status = "disabled";
+	};
+
+	lpi2c5: i2c@2D30000 {
+		compatible = "nxp,imx-lpi2c";
+		clock-frequency = <I2C_BITRATE_STANDARD>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x2D30000 0x4000>;
+		interrupts = <152 0>;
+		clocks = <&ccm IMX_CCM_LPI2C0506_CLK 0x80 24>;
+		status = "disabled";
+	};
+
+	lpi2c6: i2c@2D40000 {
```

Comment:
style nit- please use lowercase for the register address and `reg` property

Reply:
Updated.